### DISCUSSION TITLE
[Feature Request] Add multiple selection in Autocomplete component

### DISCUSSION THREAD

**Discussion by grigorzyapkov** [heroui-inc/heroui#4688] [Feature requests]
### Is your feature request related to a problem? Please describe.

The Autocomplete component doesn't support multiple items selection.

### Describe the solution you'd like

It would be nice to have multiple items selection in the autocomplete component like [in this library](https://mui.com/material-ui/react-autocomplete/#multiple-values)

### Describe alternatives you've considered

-

**Comment by derrick-null** [comment]
I have the same scenario and hope this feature can be supported soon.

**Comment by vinishanto** [comment]
I am expecting this feature

**Comment by ghost** [comment]
I need it.

**Comment by ElithPalomino** [comment]
+1

**Comment by tonkarhu** [comment]
+1

**Comment by jatin127** [comment]
+1

**Comment by duonghungkiet** [comment]
+1000000X

**Comment by Fs-gen** [comment]
i need it

**Comment by nativeanish** [comment]
need it

**Comment by impact-ls** [comment]
[I created a simple version that worked for us](https://github.com/nextui-org/nextui/issues/2299#issuecomment-2253357681)

**Comment by wingkwong** [comment]
let's track here - #4684
